cfme/utils/error.py

```diff
@@ -48,7 +48,8 @@ def _callable(f, e):
 
 def regex(expr, e):
     """Search the message of the exception using the regex expr"""
-    p = re.compile(expr)
+    escaped = re.escape(expr)
```

Comment:
expected error string has to be escaped

Reply:
**QUESTION** doesnt regex escaping defeat the purpose of passing a regex

Reply:
if string has special symbol we won't match anything.
```
my_string = "(just symbols / \ ' [ ])"
compiled = re.compile(my_string)
print compiled.search(my_string)
None
```

Reply:
the code in question is explicitly designed to match regex which is not plain strings